prefect_bitbucket/credentials.py

```diff
@@ -48,3 +71,27 @@ def _validate_username(cls, value: str) -> str:
         if not len(value) <= 30:
             raise ValueError("Username cannot be longer than 30 chars.")
         return value
+
+    def get_client(
+        self, client_type: Union[str, ClientType]
+    ) -> Union[Cloud, Bitbucket]:
+        """Get an authenticated local or cloud Bitbucket client.
+
+        Args:
+            client_type: Whether to use a local or cloud client.
+
+        Returns:
+            An authenticated Bitbucket client.
+
+        """
+        # ref: https://atlassian-python-api.readthedocs.io/
+        if isinstance(client_type, str):
+            client_type = ClientType(client_type.lower())
+        client_kwargs = dict(
+            url=self.url, username=self.username, password=self.password
+        )
+        if client_type == ClientType.CLOUD:
+            client = Cloud(cloud=True, **client_kwargs)
```

Comment:
I don't think that `cloud=True` is necessary. It looks like it gets set to `True` in `Cloud.__init__`: https://github.com/atlassian-api/atlassian-python-api/blob/master/atlassian/bitbucket/cloud/__init__.py
```suggestion
            client = Cloud(**client_kwargs)
```